hmpps-prisoner-search/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/PrisonApiServiceIntegrationTest.kt

```diff
@@ -0,0 +1,104 @@
+package uk.gov.justice.digital.hmpps.prisonersearch.search.services
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.tuple
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.prisonersearch.search.integration.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.prisonersearch.search.integration.wiremock.PrisonApiExtension.Companion.prisonApi
+
+class PrisonApiServiceIntegrationTest : IntegrationTestBase() {
+  @Nested
+  inner class GetAlerts {
+    @BeforeEach
+    fun setUp() {
+      prisonApi.stubAlertTypes(alertTypes)
+      prisonApi.stubAlertCodes(alertCodes)
+    }
+
+    @Test
+    fun `should get alerts`() {
```

Comment:
I'm not sure how far to go with negative tests for this as the data is already there and in a decent shape - though I probably should put something in - but for now I'm all happy pathed up.